docs/tutorials/T1_intro.ipynb

```diff
@@ -5,13 +5,30 @@
    "id": "6339035a",
    "metadata": {},
    "source": [
-    "# Getting started\n",
+    "# Getting started - Requirements\n",
+    "Python (🐍):  This assumes that you have a python environment installed. If you haven't, we recommend installing [Anaconda](https://docs.anaconda.com/anaconda/install/).\n",
     "\n",
```

Comment:
@emilydriano Let's change these references to the branch (below) because after we merge (SOON), we can just have folks pip install fpsim.